pages/home.vue

```diff
@@ -0,0 +1,296 @@
+        selectedName: null,
+        optionsName: [],
+        nbPerPage: 10,
+        lastId: 2147483647,
+        nbRqt: 0
+      }
+    },
+    async mounted() {
+      this.scroll()
+
+      const params = {
+        userId: this.$auth.user.id,
+        role: this.$auth.user.roleName,
+        token: this.$auth.getToken('local')
+      }
+
+      this.getGallery(params)
+
+      // fill optionNameGroup
+      this.optionsNameGroup = [
+        { value: null, text: 'Select a group name' }
+      ]
+
+      let responseDistinctNameGroup = await axiosGet({
+        axios: this.$axios,
+        url: '/api/mission/getDistinctNameGroup',
+        params
+      })
+
+      const organiseDistinctNameGroup = responseDistinctNameGroup.data.nameGroups
+        .map(elem => {
+          return {text: elem.nameGroup, value: elem.nameGroup}
+        })
+        .filter(elem => elem.text)
+
+      this.optionsNameGroup = this.optionsNameGroup.concat(organiseDistinctNameGroup)
+      // END fill optionNameGroup
+
+
+      // fill optionNameClient
+      this.optionsNameClient = [
+        { value: null, text: 'Select a client name' }
+      ]
+
+      let responseDistinctNameClient= await axiosGet({
+        axios: this.$axios,
+        url: '/api/mission/getDistinctNameClient',
+        params
+      })
+
+      const organiseDistinctNameClient = responseDistinctNameClient.data.nameClients
+        .map(elem => {
+          return {text: elem.nameClient, value: elem.nameClient}
+        })
+        .filter(elem => elem.text)
+
+      this.optionsNameClient = this.optionsNameClient.concat(organiseDistinctNameClient)
+      // END fill optionNameClient
+
+
+      // fill optionName
+      if (this.$auth.user.roleName === 'ADM') {
+        this.optionsName = [
+          {value: null, text: 'Select a name'}
+        ]
+
+        let responseDistinctName = await axiosGet({
+          axios: this.$axios,
+          url: '/api/mission/getDistinctName',
+          params
+        })
+
+        const organiseDistinctName = responseDistinctName.data.names
+          .map(elem => {
+            return {text: elem.name, value: elem.name}
+          })
+          .filter(elem => elem.text)
+
+        this.optionsName = this.optionsName.concat(organiseDistinctName)
+      }
+      // END fill optionNameClient
+    },
+    methods: {
+      async updateFilters() {
+        const params = {
+          validated: this.selectedValidated,
+          nameGroup: this.selectedNameGroup,
+          nameClient: this.selectedNameClient,
+          name: this.selectedName,
+          userId: this.$auth.user.id,
+          role: this.$auth.user.roleName,
+          token: this.$auth.getToken('local'),
+          nbPerPage: this.nbPerPage,
+          lastId: 2147483647
+        }
+
+        let gallery = await axiosGet({
+          axios: this.$axios,
+          url: '/api/mission/galery',
+          params
+        })
+
+        if (gallery.data.mission) {
+          this.nbRqt = 1
```

Comment:
Maybe we could rename it so it's more clear what it is ? numberOfRequest for example ?